tasks/run-matlab-tests/v0/task.json

```diff
@@ -97,6 +101,42 @@
             "groupName": "testArtifacts",
             "required": false,
             "helpMarkDown": "Path to write test results report in JUnit XML format."
+        },
+        {
+            "name": "useParallel",
+            "type": "boolean",
+            "label": "Use Parallel",
+            "defaultValue": false,
+            "required": false,
+            "groupName": "runCustomization",
+            "helpMarkDown": "Whether to run tests in parallel. This feature requires a Parallel Computing Toolbox license, and might not be compatible with other options. If incompatible, testing occurs in serial regardless of the value of this feature."
```

Comment:
In Azure Pipelines, we use "argument" instead of "option." Suggesting some changes:

Whether to run tests in parallel (requires Parallel Computing Toolbox). This feature might not be compatible with certain arguments, in which case, tests run in serial regardless of the specified value.